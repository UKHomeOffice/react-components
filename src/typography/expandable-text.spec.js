import React from 'react'
import { render } from 'enzyme'
import ExpandableText from 'govuk-react-components/typography/expandable-text';

describe('The Expandable Text Component', () => {

  it('should render correctly', () => {
    expect(
      render(<ExpandableText title="Lorem ipsum">Dolor set amet</ExpandableText>)
    ).toMatchSnapshot()
  });

})