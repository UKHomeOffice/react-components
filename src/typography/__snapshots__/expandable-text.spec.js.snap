// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Expandable Text Component should render correctly 1`] = `
<details>
  <summary>
    <span
      class="summary"
    >
      Lorem ipsum
    </span>
  </summary>
  <div
    class="panel panel-border-narrow"
  >
    Dolor set amet
  </div>
</details>
`;
